lib/util/JSON.lua

```diff
@@ -0,0 +1,21 @@
+local exists, json = pcall(require, "dkjson")
+
+local JSON = {}
+
+function JSON.Encode(input)
```

Comment:
Can we rename this file to `lib/json.lua` and change the casing of the methods to be camelCase? All of the APIs we emulate are PascalCase, but the internal utilities and files in this project are camelCase.